moveit_ros/moveit_servo/CMakeLists.txt

```diff
@@ -202,13 +184,33 @@ if(BUILD_TESTING)
 
 endif()
 
+ament_export_include_directories(include)
+ament_export_libraries(
+  ${SERVO_LIB_NAME}
+  ${SERVO_PARAM_LIB_NAME}
```

Comment:
Why not export `COMPOSABLE_NODE_NAME` too? This is the library that provides a ROS node component

Reply:
The way I'm using it here is even wrong (these shouldn't be library names).  Here is the issue I created about my confusion around how to use this and the official documentation.  https://github.com/ros2/ros2_documentation/issues/1094

The short of it is this line should not be here and the correct (as far as I understand now) should be this instead:

```cmake
# Install Targets
install(
  TARGETS
    ${SERVO_LIB_NAME}
    ${SERVO_PARAM_LIB_NAME}
    ${COMPONENT_NODE_NAME}
    ${SERVO_CONTROLLER_INPUT}
    ${CPP_DEMO_NAME}
    ${FAKE_SERVO_CMDS_NAME}
  EXPORT
    export_${PROJECT_NAME}
  ARCHIVE DESTINATION lib
  LIBRARY DESTINATION lib
  RUNTIME DESTINATION bin
  INCLUDES DESTINATION include
)

# This calls the install(EXPORT ...) cmake functions to export the target libraries
# so other pojects can depend on libraries from moveit_servo.
ament_export_targets(export_${PROJECT_NAME} HAS_LIBRARY_TARGET)
ament_export_dependencies(${THIS_PACKAGE_INCLUDE_DEPENDS})
```